bin/run_docker.py

```diff
@@ -117,26 +118,42 @@ def create_log_file(
     If no text is given, it writes "No Logs" to the file.
 
     Arguments:
-        log_filename: The name of the log file to create
-        log_text: The text to write to the log file. If given as a byte string,
-                  it will be decoded as UTF-8 before being written.
+        log_file_name: The name of the log file to create
+        log_max_size: The maximum size of the log file in kilobytes
+        log_file_path: The path where the log file will be created.
+                       If not specified, the current working directory will be used.
+        log_text: The text to write to the log file
 
     """
-    if not log_file_path:
+    if log_file_path is None:
         log_file_path = os.getcwd()
 
+    if log_text is None:
```

Comment:
nit. Could reduce the cognitive complexity by setting this default in the arguments:
```
def create_log_file(
    log_file_name: str,
    log_max_size: int,
    log_file_path: Optional[Union[None, str]] = None,
    log_text: Optional[Union[str, bytes]] = "No Logs",
) -> None:
.
.
.
.
    if isinstance(log_text, bytes):
        log_text = log_text.decode("utf-8")
```

Reply:
I see what you mean. This should be fine then. I suppose if we did want to reduce this the code that calls this would need to pass along not `None`, but no argument at all.